packages/react-select/src/Select.tsx

```diff
@@ -628,13 +630,28 @@ export default class Select<
     this.instancePrefix =
       'react-select-' + (this.props.instanceId || ++instanceId);
     this.state.selectValue = cleanValue(props.value);
+
+    // If `value` or `defaultValue` props are not empty then announce them
+    // when the Select is focused
+    if (this.state.selectValue.length) {
+      this.state.ariaSelection = {
+        value: this.state.selectValue as OnChangeValue<Option, IsMulti>,
```

Comment:
I'm not sure we should be okay with casting this. `selectValue` is an array, but the `value` property in this context is referring to either an array for multi-select or a single value (or `null`) for single-select. I think we probably want to use what we use elsewhere:

```
value: valueTernary(
  this.props.isMulti,
  this.state.selectValue,
  this.state.selectValue[0] || null
);
```

Reply:
I have refactored a bit and am using this pattern now